widget/richtext.go

```diff
@@ -89,19 +88,18 @@ func (t *RichText) CreateRenderer() fyne.WidgetRenderer {
 // MinSize calculates the minimum size of a rich text widget.
 // This is based on the contained text with a standard amount of padding added.
 func (t *RichText) MinSize() fyne.Size {
-	// we don't return the minCache here, as any internal segments could have caused it to change...
 	t.ExtendBaseWidget(t)
 
-	min := t.BaseWidget.MinSize()
-	t.minCache = min
-	return min
+	// We don't return the cached value as any internal segments could have caused it to change...
```

Comment:
We should likely fix this as it kind of goes against the idea of refreshing a widget when it changes state. SOunds like a bug but it isn't new behaviour to this PR so I'll let it slide for now.